shopfloor_delivery_shipment_mobile/static/wms/src/scenario/delivery_shipment.js

```diff
@@ -82,7 +82,6 @@ const DeliveryShipment = {
                 <v-card-title>Are you sure you want to close the shipment ?</v-card-title>
             </v-card>
             <item-detail-card v-if="state_is('validate')"
-                :key="make_state_component_key(['lading-detail', 123])"
```

Comment:
I didn't say you don't need any key, I said you don't need "123" as lon as it's a fixed string does not change anything. If you have a real ID of a record just use that instead.

Reply:
Ok, I removed the 123 but did not add an id. I am not sure it is a useful key.

Reply:
it is still useful because otherwise 2 elements of the same type will have the same key. In this way each of them will have its specific one.